src/video/damage_tracker.h

```diff
@@ -5,14 +5,21 @@
 #include <stdbool.h>
 
 typedef struct damage_tracker {
-    vga_index dirty_range_start;
-    vga_index dirty_range_end;
+    vga_index dirty_range_first;
+    vga_index dirty_range_last;
```

Comment:
Are the first and last names still relevant ?

Reply:
damgae_tracker uses an inclusive range, so I am intentionally renaming its internals to be first/last.

Reply:
Hmmh, that's an annoying remnant. I've tried to use for-loop friendly ranges everywhere, but this one probably slipped through the net. Okay, let's keep it inclusive, though it might make sense to change to exclusive range (like we do elsewhere in the code) in the future.